social_protection/services.py

```diff
@@ -59,6 +59,29 @@ def update(self, obj_data):
     def delete(self, obj_data):
         return super().delete(obj_data)
 
+    def _data_for_json_ext(self, obj_data):
+        beneficiary = Beneficiary.objects.get(id=obj_data.get("id"))
+        individual = beneficiary.individual
+        benefit_plan = beneficiary.benefit_plan
+        individual_identity_string = f'{individual.first_name} {individual.last_name}'
+        json_ext_data = {"individual_identity": individual_identity_string,
```

Comment:
I'm not sure whether provided structure could affect on custom filters selection with adding two additional redundant field to filter.
Currently after beneficiary upload the json ext structure with addtional data is flatten, Maybe we can add additional 'key' with nested those values? To be checked.

Reply:
@jdolkowski We don't need to care about that approach because until we don't have that fields on schema level everything should work as expected.